File: tests/expected/print.nim
proc show() =
  echo "b"
  echo 2, " ", "b"
  let a: float64 = 2.1
  echo a
  let b = 2.1
  echo b
  let c = true
  echo if c: "True" else: "False"

proc main() =
  show()

main()

File: tests/expected/julia/print.jl

function show()
    println("b")
    println("$(2)b")
    a::Float64 = 2.1
    println(a)
    b = 2.1
    println(b)
    c = true
    println(c ? ("True") : ("False"))
    name = "test"
    val = true
    write(sys.stdout, "$(name)_vtables_dispatch_ = $(val)")
end

if abspath(PROGRAM_FILE) == @__FILE__
    show()
end
